package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- Hey, the app won't run because it's Missing the `start` property from your script, please, add the

` "start": "webpack serve --open"` to fix this. 
- Install the `npm i webpack-dev-server` to start the server


NOTE: *After doing those, your server will start, but it won't show up UI because, you've not configured your webpack correctly. The snippet of code I'd add here is to just make you see your UI. Please, kindly revisit the microverse webpack activity to setup your webapack correctly.*